admin-ui/app/utils/UploadSSA.js

```diff
@@ -0,0 +1,110 @@
+import React, { useCallback, useContext, useEffect, useState } from 'react'
+import { Container } from 'Components'
+import { useDropzone } from 'react-dropzone'
+import logo from 'Images/logos/logo192.png'
+import { Button } from 'reactstrap'
+import { ThemeContext } from 'Context/theme/themeContext'
+import applicationStyle from 'Routes/Apps/Gluu/styles/applicationstyle'
+import { useTranslation } from 'react-i18next'
+import { useSelector, useDispatch } from 'react-redux'
+import { uploadNewSsaToken } from '../redux/actions'
+import './LicenseScreens/style.css'
+function UploadSSA() {
+  const dispatch = useDispatch()
+  const { t } = useTranslation()
+  const theme = useContext(ThemeContext)
+  const selectedTheme = theme.state.theme
+  const isLoading = useSelector((state) => state.licenseReducer.isLoading)
+
+  const [selectedFileName, setSelectedFileName] = useState(null)
+  const [selectedFile, setSelectedFile] = useState(null)
+    const [jwt, setJWT] = useState(null)
+
+
+  const readJWTFile = () => {
+    const reader = new FileReader()
+
+    reader.onload = () => {
+      const token = reader.result
+      setJWT(token)
+    }
+
+    const blob = new Blob([selectedFile])
+    reader.readAsText(blob)
+  }
+
+  useEffect(() => {
+    if(selectedFile){
+        readJWTFile();
+    }
+  },[selectedFile])
+
+
+  const onDrop = useCallback((acceptedFiles) => {
+    const file = acceptedFiles[0]
+    setSelectedFileName(file.name)
+    setSelectedFile(file)
+  }, [])
+  const {
+    getRootProps: getRootProps1,
+    getInputProps: getInputProps1,
+    isDragActive: isDragActive1,
+  } = useDropzone({
+    onDrop: onDrop,
+    accept: {
+      'application/jwt': ['.jwt'],
+    },
+  })
+
+  const submitData = () => {
+    if(selectedFile){
+        dispatch(uploadNewSsaToken({licenseKey:jwt}))
```

Comment:
`jwt` is `ssa`. The `licenseKey` name should be `ssa`.

Reply:
Changed as suggested